fudge-ratio/src/main/webapp/about/index.html

```diff
@@ -1,27 +1,33 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <link rel="stylesheet" href="about.css" />
-        <link href="mobile.css" rel="stylesheet" media="screen and (max-width: 1100px)">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" charset="UTF-8" />
+        <meta name="theme-color" content="#5D7F95">
+        
+        <link rel="stylesheet" href="/styles/general.css" />
+        <link rel="stylesheet" href="/styles/header.css" />
+        <link rel="stylesheet" href="/styles/about.css" />
+
+        <link rel="stylesheet" href="desktop.css" media="screen and (min-width: 700px)" />
+        <link href="mobile.css" rel="stylesheet" media="screen and (max-width: 700px)">
+
         <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-        <meta charset="UTF-8" name="viewport" content="width=device-width, initial-scale=1.0">
         <title>About Us</title>
     </head>
     <body>
-        <div class="banner">
-            <h1 id="logo">Fudge Ratio Calculator</h1>
+        <header>
+            <h1>Fudge Ratio Calculator</h1>
             <div id="links">
-                <a href="../">Home</a>
-                <a href="/calculator">Calculator</a>
+                <a href="/">Home</a>
```

Comment:
has the structure of the directories change?

Reply:
I think this PR is missing the new changes from master. A rebase of this branch with the master branch will solve this.

Reply:
* No, the structure has not changed, the 2 expressions are equivalent:
   - ``../`` means, do to the index file in the parent folder 
   - ``/`` means, do to the index file in the ROOT folder 

Turns out that the parent folder is the root folder, but that is not a given, I think home will always be at ``/``, so with this change we can move this file without worrying about relative directions

Reply:
Also, I did not a rebase since I am the only one that touched the about folder in the last couple of days, maybe you are confusing this with the changes I did in the landing page.

But nevertheless the point is correct, I need to change the routes to work on the live server

Reply:
Oh, I understand. Can you commit the changes of the routes?

Reply:
I did the rebase ... and you was totally correct

Reply:
![Upss](https://media.giphy.com/media/WsGKzpdvWXY7jBNLqc/giphy.gif)

Reply:
I am, I was the only one that touched about folder but I based this PR on top a branch that did some changes to that folder (so to recap, that change was proposed and intended to be review here: https://github.com/gilemos/FudgeRatio/pull/32/commits/b5b916bf0dbcabf2561b827b539d81cba51beee6), now, thanks to the rebase, we can see that this PR is now 2 simple commits with 43 lines